kinoml/core/proteins.py

```diff
@@ -109,9 +109,17 @@ class ProteinStructure(BaseProtein, BaseStructure):
     def from_file(cls, path, ext=None, **kwargs):
         from MDAnalysis import Universe
         from pathlib import Path
+
+        identifier = f"{path.split('/')[-1].split('.')[0]}"  #  set id to be the file name
+
         u = Universe(path)
         p = Path(path)
-        return cls(name=p.name, metadata={"path": path}, universe=u, **kwargs)
+        return cls(
+            pname=p.name,
+            metadata={"path": path, "id": identifier},
+            universe=u,
+            **kwargs,
+        )
```

Comment:
I think we are using different Black settings, right? I was reformatting with `-l 99`, but I've seen Talia is also doing it with defaults, so we might as well just use default configurations.

Reply:
Yes, I've just been running Black with defaults!